tests/integration/tests/test_util/util.py

```diff
@@ -209,7 +209,7 @@ def preload_snaps(instance: harness.Instance):
             source=f"/tmp/{snap_file}",
             destination=remote_path,
         )
-        instance.exec(["snap", "install", remote_path])
+        util.stubbornly(retries=3, delay_s=5).on(instance).exec(["snap", "install", remote_path])
```

Comment:
i think we're IN the `util` module

```suggestion
        stubbornly(retries=3, delay_s=5).on(instance).exec(["snap", "install", remote_path])
```